### DISCUSSION TITLE
升级到4.0.87后, less里的url()报错

### DISCUSSION THREAD

**Discussion by skyblue** [umijs/umi#11795] [General]
@umijs/max 从4.0.86升级到4.0.87 后
overrides.less里使用的
`background-image: url('assets/bg.png');`
报错
`Module not found: Error: Can't resolve 'assets/bg.png' in '/src/.umi/core'`

**Comment by fz6m** [comment]
先这么用：

```css
  background-image: url('~@/assets/bg.png');
```

后续在 #11800 里跟进。
